analyses/mutational-signatures/run_mutational_signatures.sh

```diff
@@ -32,5 +32,11 @@ QUICK_MUTSIGS=$ABBREVIATED_MUTSIGS ANALYSIS=1 bash 03-de_novo_range_of_nsignatur
 # Process results from de novo extraction 
 Rscript -e "rmarkdown::render('04-analyze_de_novo.Rmd', clean = TRUE, params = list(is_ci = ${ABBREVIATED_MUTSIGS}))"
 
-
-# Next steps: Fitting the 8 known CNS signatures
\ No newline at end of file
+# Fit the 8 known CNS signatures
+Rscript --vanilla 05-fit_cns_signatures.R  \
+  --abbreviated $ABBREVIATED_MUTSIGS
+  
+# Process, visualize the 8 known CNS signature exposures
+Rscript -e "rmarkdown::render('06-analyze_cns_fit.Rmd', clean = TRUE)"
+
+  
```

Comment:
This is very strange?

Reply:
What is strange? I'm not seeing it...

Reply:
There's a bunch of whitespace but the little ⛔ indicates there is no new line at the end.

Reply:
Oh I see, that IS weird..... Hmm.  I'll poke around.